iam-api/pom.xml

```diff
@@ -69,40 +62,23 @@
     </dependencies>
 
     <build>
-        <resources>
```

Comment:
nit: just curious, but why don't we need this _resources_ block anymore? What was it for? I mean here and in all other places